### ISSUE TITLE
KICS install.sh script cannot completed on v1.5.1 and and v1.5.2

### ISSUE THREAD

**Issue by adhitris** [Checkmarx/kics#4871]
Hi, I tried to run KICS install using One-liner Install Script using Curl command:
`sudo curl -sfL 'https://raw.githubusercontent.com/Checkmarx/kics/master/install.sh' | bash -s -- -b /usr/local/bin
`
I tried with debugging mode and it return:

38 | Checkmarx/kics debug http_download https://github.com/Checkmarx/kics/releases/v1.5.2
39 | Checkmarx/kics info found version: 1.5.2 for v1.5.2/linux/x64
40 | Checkmarx/kics debug downloading files into /tmp/tmp.rgij4CjclP
41 | Checkmarx/kics debug http_download https://github.com/Checkmarx/kics/releases/download/v1.5.2/kics_1.5.2_linux_x64.tar.gz
42 | Checkmarx/kics debug http_download https://github.com/Checkmarx/kics/releases/download/v1.5.2/kics_1.5.2_checksums.txt
43 | % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
44 | Dload  Upload   Total   Spent    Left  Speed
45 |  
46 | 0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
47 | 100  1700  100  1700    0     0  47222      0 --:--:-- --:--:-- --:--:-- 47222

And when I tried the KICS scanning command:
` kics scan -p`

it gave kics: command not found error response on Linux based CLI

But when I tried to add tag using v1.5.0 , there is no issue and KICS run properly.

**Comment by rafaela-soares** [comment]
Hello, @adhitris 🙂 Thank you so much for contacting us!

KICS deprecated the availability of binaries in the GitHub releases assets as of version 1.5.2.

The lastest binary available is 1.5.1: https://github.com/Checkmarx/kics/releases/tag/v1.5.1

**Comment by rafaela-soares** [comment]
Hello, @adhitris!

We will close this issue. The file `install.sh` will be probably removed in the next KICS release (1.6.4) since KICS deprecated the availability of binaries. However, we documented how to build KICS from source code (PR #5982).

```sh
cd kics
go mod vendor
LINUX/MAC: go build -o ./bin/kics cmd/console/main.go
WINDOWS: go build -o ./bin/kics.exe cmd/console/main.go (make sure to create the bin folder)
```
